ui/app/rif/constants/index.js

```diff
@@ -36,4 +36,9 @@ export const luminoNodeIconProps = {
 export const rifStorageIconProps = {
     color: '#AD3232',
     icon: faArchive,
-}
\ No newline at end of file
+}
+
+export const registrationTimeouts = {
```

Comment:
Can you please put some comments to reflect what those numbers means?

Reply:
Sure